### ISSUE TITLE
asar option doesn't put anything in app.asar

### ISSUE THREAD

**Issue by rahatarmanahmed** [maxogden/electron-packager#85]
Running Windows x64
Building win32-x64
electron-packager v4.1.3
Options:
```javascript
// My script handles arguments, but I tested with the following values.
var platform = 'win32';
var arch =  'x64'
var packagerOpts = {
	dir: __dirname + '/..',
	name: 'Squelch-' + platform + '-' + arch,
	asar: true,
	version: pkg.devDependencies['electron-prebuilt'].match(/\d+\.\d+\.\d/)[0],
	platform: platform,
	arch: arch,
	out: __dirname + '/../release',
	prune: true,
	ignore: [
	'/test($|/)',
	'/tools($|/)',
	'/scripts($|/)',
	'/release($|/)',
	]
};
packager(packagerOpts, function(err, path) {
	if(err) { throw err; }
	console.log('Successfully built package at ' + path);
});
```
The build completes without errors, but my app.asar seems to be empty. I can run `asar list resources/resources/app.asar` and it won't output anything. My app code is there in `resources/app`, and removing that makes the app open the default_app instead, so it's definitely not loading from app.asar.

...Actually, why is my app code there as a folder? Either the asar option doesn't work right or that just ruins the point of asar.

**Comment by kfranqueiro** [comment]
This sounds like #81 and #73 which have been resolved on master (but not in a release yet).

**Comment by kfranqueiro** [comment]
I believe this is fixed in 4.2.0 and 5.0.0 so I'm closing this.  Let us know if you still experience this issue after updating.  Thanks!
